lms/static/js/components/header/views/header.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * A generic header view class.
+ */
+;(function (define) {
+    'use strict';
+    define(['backbone', 'text!templates/components/header/header.underscore'],
+           function (Backbone, headerTemplate) {
+               var HeaderView = Backbone.View.extend({
+                   initialize: function (options) {
+                       this.template = _.template(headerTemplate);
+                       this.listenTo(this.model, 'change', this.render);
+                       this.render();
+                   },
+
+                   render: function () {
+                       var json = this.model.toJSON();
```

Comment:
Must: change this to use `this.model.attributes` rather than `toJSON`. From what I've learned, Backbone's conventions are to use `toJSON` for serializing a model to it's server representation, and to use `attributes` for just getting the Backbone-land-named model attributes. This makes Backbone flexible for cooperating with weird APIs.

[Source](http://stackoverflow.com/a/16489929)